examples/multigpu_advtrain/target_attack.py

```diff
@@ -0,0 +1,257 @@
+import numpy as np
+import tensorflow as tf
+
+from cleverhans.model import Model, CallableModelWrapper
+from cleverhans.attacks import Attack, FastGradientMethod
+from cleverhans.utils_tf import model_loss
+
+from model import clone_variable, MLP_probs
+
+
+class ProjectedGradientDescentMethod(Attack):
+
+    """
```

Comment:
The docstring and the class name are out of sync

Reply:
Fixed in 460d3f3.